<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetDialog</class>
 <widget class="QDialog" name="ResetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lab_user">
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="user_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lab_email">
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>邮箱：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="email_edit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lab_verifycode">
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>验证码：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="verifycode_edit"/>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="TimerBtn" name="btn_verifycode">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>获取</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lab_pwd">
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>新密码：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="newpwd_edit"/>
    </item>
    <item row="3" column="4">
     <widget class="ClickedLabel" name="newpwd_visible">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="btn_sure">
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>122</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QPushButton" name="btn_return">
      <property name="font">
       <font>
        <family>方正水黑简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lab_error_tip">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正水黑简体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>错误提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_photo">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>4</y>
     <width>391</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickedLabel</class>
   <extends>QLabel</extends>
   <header>clickedlabel.h</header>
  </customwidget>
  <customwidget>
   <class>TimerBtn</class>
   <extends>QPushButton</extends>
   <header>timerbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
